### PR TITLE
measure speed as average of values

### PR DESCRIPTION
**Pull request by idruzhitskiy**

### CODE REVIEW

File: src/bin/bobp.rs [hunk 1]
```diff
@@ -276,22 +278,24 @@ fn stat_worker(
 
         let put_error = stat.put_error.load(Ordering::Relaxed);
         let get_error = stat.get_error.load(Ordering::Relaxed);
-
+        let put_spd = put_count_spd as f64 * k;
+        let get_spd = get_count_spd as f64 * k;
+        put_speed_values.push(put_spd);
```
Comment (agend) [inline src/bin/bobp.rs]: how large could this `vec` get?

Comment (idruzhitskiy) [reply src/bin/bobp.rs]: len = execution time in seconds

File: src/bin/bobp.rs [hunk 2]
```diff
@@ -318,6 +320,10 @@ fn stat_worker(
     );
 }
 
+fn average(values: &[f64]) -> f64 {
+    finite_or_default(values.iter().sum::<f64>() / values.len() as f64)
```
Comment (agend) [inline src/bin/bobp.rs]: better to have average in a window. @pyakushin

Comment (pyakushin) [reply src/bin/bobp.rs]: It can be both. Average on all values for results and moving average for current statistics

Comment (pyakushin) [reply src/bin/bobp.rs]: It is final stats, so I think it is fine

**Pull request merged**